website/src/index.css

```diff
@@ -7,9 +8,17 @@
   --black: hsla(0deg 0% 0% / 100%);
 }
 
+html {
+  min-height: 100%;
+}
+
 body {
   font-family: Roboto, sans-serif;
   font-size: 16px;
+  height: 100%;
+  padding-bottom: 3em;
+  display: flex;
```

Comment:
why is flex needed here?

Reply:
I was trying something out and forgot to delete but it does nothing. I removed it.